tests/providers/amazon/aws/sensors/test_batch.py

```diff
@@ -44,25 +43,25 @@ def setUp(self):
     @mock.patch.object(BatchClientHook, "get_job_description")
     def test_poke_on_success_state(self, mock_get_job_description):
         mock_get_job_description.return_value = {"status": "SUCCEEDED"}
-        self.assertTrue(self.batch_sensor.poke({}))
+        assert self.batch_sensor.poke({})
         mock_get_job_description.assert_called_once_with(JOB_ID)
 
     @mock.patch.object(BatchClientHook, "get_job_description")
     def test_poke_on_failure_state(self, mock_get_job_description):
         mock_get_job_description.return_value = {"status": "FAILED"}
-        with self.assertRaises(AirflowException) as e:
+        with pytest.raises(AirflowException) as ctx:
             self.batch_sensor.poke({})
 
-        self.assertEqual("Batch sensor failed. AWS Batch job status: FAILED", str(e.exception))
+        assert "Batch sensor failed. AWS Batch job status: FAILED" == str(ctx.value)
```

Comment:
You could use `match` argument in `pytest.raises` for check exception error message, [some sample](https://github.com/Taragolis/airflow/blob/6ecb2f8cc4a949284e3143796ee5c4097bde1c25/tests/providers/amazon/aws/sensors/test_batch.py#L51-L54)